src/managed_scip.jl

```diff
@@ -15,6 +15,8 @@ mutable struct ManagedSCIP
     conss::Dict{ConsRef, Ref{Ptr{SCIP_CONS}}}
     var_count::Int64
     cons_count::Int64
+    conshdlrs::Dict{Any, Ptr{SCIP_CONSHDLR}}
+    conshdlrconss::Dict{Any, Ptr{SCIP_CONS}}
```

Comment:
Are these dict indexed by anything or by a string / int / something?

Reply:
Well, they are indexed by the user-defined types, so, subtypes of `SCIP.AbstractConstraintHandler` or `SCIP.AbstractConstraint{CH}` respectively. But it can be mixed, so I don't think it's possible to use a type parameter.